lib/do_instance/do_create.js

```diff
@@ -496,6 +498,19 @@ do_create.options = [
             'This option can be used multiple times for multiple NICs. ' +
             'Valid keys are: ' + Object.keys(NETWORK_OBJECT_FIELDS).join(', ')
     },
+    {
+        names: ['delegate-dataset'],
+        type: 'bool',
+        help: 'Create a Delegated Dataset on this instance. If set a ' +
+              'delegated dataset is created, though the instance is ' +
+              'prevented from receiving ZFS datasets (zfs recv). Note ' +
+              'that Triton CloudAPI instances must have the SAPI config ' +
+              'value "experimental_cloudapi_delegate_dataset=true" ' +
+              'for create requests using this option to be allowed.',
+        // mark this option hidden for now, see TRITON-853, at least until we
+        // get a full implementation of RFD 044.
+        hidden: true
```

Comment:
Just checking - is this still the case (that this option should be hidden)?

Reply:
I have no strong opinion about it. I'm ok with it not being hidden, but others may feel differently.

Reply:
Remove hidden - CloudAPI will have it exposed, so node-triton might as well too.

Reply:
Done and done.